subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -23,10 +23,6 @@
 
```

Comment:
Should we include the defs file here, too?

Really on the fence about this once, since it already includes "nrf_cloud_codec"

Reply:
add it just for visibility?  i have no preference.

Reply:
Yeah, just for visibility -- same with the other place I suggested it. But I don't feel particularly strongly about it

Reply:
ok, i'll just leave it out since it is not necessary.